app/src/components/Card/Sections/Attachments.tsx

```diff
@@ -1,21 +1,94 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Modal, Image } from '@mantine/core'
 import { IFile } from '../../../types/File'
 import { isEmpty } from 'lodash'
+import {
+  RiArrowLeftSLine,
+  RiArrowRightSLine,
+  Minus,
+  Plus,
+  DownloadIcon,
+} from '../../Project/Content/IconsCard'
+import { useDisclosure } from '@mantine/hooks'
 
-function FileTile({ file }: { file: IFile }) {
-  switch (file.type) {
```

Comment:
Actually this component won't use only for images, and I don't get it why you change this code 🙃